apps/site/assets/ts/helpers/fetch.ts

```diff
@@ -7,16 +7,23 @@ export type fetchAction =
 export interface State {
   // @ts-ignore should add a generic
   data: any | null; // eslint-disable-line
+  initial: any | null; // eslint-disable-line
```

Comment:
It's a new prop that is meant to store the first result of data. It's only utilized for a specific use case - the upcoming departures - so that the upcoming departures component doesn't re-render when `data` changes, which was happening before, since we changed upcoming departures to only look at `initial`.